src/components/AccountForms.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="q-pa-md">
+    <q-card class="my-card">
+      <q-card-section>
+        <div class="text-h6 row items-center justify-center">Welcome to the store!</div>
+      </q-card-section>
+      <q-tabs v-model="tab" class="text-teal" dense align="justify" :breakpoint="0">
+        <q-tab label="Create account" name="one" />
+        <q-tab label="Have an account?" name="two" />
+      </q-tabs>
+      <q-separator />
+      <q-tab-panels v-model="tab" animated>
+        <q-tab-panel name="one">
+          <q-form class="q-gutter-md row items-center justify-center">
+            <q-input square filled clearable v-model="email" type="email" label="Email" class="q-pa-sm col-12" />
+            <q-input square filled clearable v-model="fullName" type="text" label="Full Name" class="q-pa-sm col-12" />
+            <q-input square filled clearable mask="(###) ### - ####" unmasked-value v-model="phoneNumber"
+              label="Phone number" class="q-pa-sm col-12" />
+            <q-input square filled clearable v-model="password" type="password" label="Password"
+              class="q-pa-sm col-12" />
+            <q-input square filled clearable v-model="address" type="address" label="Address" class="q-pa-sm col-12" />
+            <q-btn unelevated color="light-green-7" size="md" class="q-pa-sm col-12" label="Sign Up" @click="signUp" />
+            <q-checkbox left-label v-model="remainSignedIn" label="Stay logged in" checked-icon="task_alt"
+              unchecked-icon="highlight_off" />
+          </q-form>
+        </q-tab-panel>
+        <q-tab-panel name="two">
+          <q-form class="q-gutter-md row items-center justify-center">
+            <q-input square filled clearable v-model="email" type="email" label="email" class="q-pa-md col-12" />
+            <q-input square filled clearable v-model="password" type="password" label="password"
+              class="q-pa-md col-12" />
+            <q-btn unelevated color="light-green-7" size="md" class="q-pa-md col-12" label="Login" @click="login" />
+            <q-checkbox left-label v-model="remainSignedIn" label="Stay logged in" checked-icon="task_alt"
+              unchecked-icon="highlight_off" />
+          </q-form>
+        </q-tab-panel>
+      </q-tab-panels>
+    </q-card>
+  </div>
+</template>
+<script setup>
+import { ref } from 'vue'
+import useHttpQuery from 'src/compositionFunctions/useHttpQuery'
+import useLocalStorage from 'src/compositionFunctions/useLocalStorage'
+import { useRouter } from 'vue-router'
+const { saveUserData } = useLocalStorage()
+const { addNewUser, verifyUserCredentials } = useHttpQuery()
+const router = useRouter()
+const tab = ref('one')
+const email = ref('')
```

Comment:
You can simplify the writing by using js objects:
const loginData = ref({}) OR
const loginData = ref({ email: '', fullName: '', ... }) if you want to specifically instantiate the values